.travis.yml

```diff
@@ -137,16 +137,16 @@ matrix:
           env: NUMPY_VERSION=1.15
 
         # Do a PEP8 test with flake8
-        - os: linux
-          stage: Initial tests
-          env: MAIN_CMD='flake8 {{ cookiecutter.module_name }} --count --show-source --statistics $FLAKE8_OPT' SETUP_CMD=''
+        # - os: linux
+        #  stage: Initial tests
+        #  env: MAIN_CMD='flake8 marxs --count --show-source --statistics $FLAKE8_OPT' SETUP_CMD=''
 
     allow_failures:
         # Do a PEP8 test with flake8
         # (do allow to fail unless your code completely compliant)
-        # - os: linux
-        #   stage: Initial tests
-        #   env: MAIN_CMD='flake8 {{ cookiecutter.module_name }} --count --show-source --statistics $FLAKE8_OPT' SETUP_CMD=''
+        - os: linux
```

Comment:
for allow_failures it needs to be exactly duplicated jobs from the matrix above. 

Also, where did you copy the template? there shouldn't have been there any cookiecutter variables in there to begin with.

Reply:
I used the templete some time ago (before cookiecutter). Now, I found that a lot of things were out of date, for example, astropy gave me 6 deprecation warning when running tests, all realted to my setup.py. So, I decided to fix those as far as I can and updated `.travis.yml`, `setup.py` and `setup.cfg` manually by copying over the relevant lines from the current template in the cookie-cutter branch, but I did not replace all the variables. If I had taken the master branch, it would have said "PACKAGENAME" instead, I guess.

I'm not doing that now, but if you look at the setup.py you would see that I'm still using the old scheme where we manually read the requirements, name, etc. from setup.cfg instead of having setuptools take care of that. I just could not figure out how to do my optional cffi dependency in there. (Note that this is not a request for help. I'll just leave it as is is for now. I might split out the cffi bit and binding to an external C library into a separate package anyway or replace it with python / cython inside this package. I'm just explaining this, since you asked for feedback on how the infrastructure packages are used and what possible problems might look like. Despite the complexity I think it's great to have e.g. the template so that I can copy a relatively modern setup [in term of python versions, numpy dependencies etc.] instead of figuring that all out from scratch.)

Thanks a lot again for your help!